tests/helper/sshclient.py

```diff
@@ -249,6 +257,8 @@ def execute_command(self, command: str, timeout: int = 30, quiet: bool = False)
             self.client = self.__connect()
         if not quiet:
             logger.info(f"$ {command.rstrip()}")
+        if self.sudo and not disable_sudo:
+            command = 'sudo ' + command
```

Comment:
I'd rather write `command = f'sudo {command}'` but this also works. No showstopper...